jbehave-support-core-test/jbehave-support-core-test-app/src/main/resources/static/title.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML>
```

Comment:
Nitpicking: maybe change the name from title.html to something like frame.html or iframe.html?

Reply:
renamed to frames.html and title changes to Marvelous Frames